pkgs/development/perl-modules/Bio-BigFile/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, perl, buildPerlModule, fetchpatch, fetchurl, fetchFromGitHub, kent,
+  BioPerl, IOString, libpng, libmysqlclient, openssl }:
+# This version of bio::DB::bigfile is suited for ENSEMBL VEP
+# See https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile
+
+# Only kent 335 works with Bio-BigFile. The kent package on nix must be slightly changed
+# 1. building utils fails (see patche below)
+# 2. we need headers (PR submitted for kent package)
+let
+  kent335 = kent.overrideAttrs (old: rec {
+    pname = "kent";
+    version = "335";
+
+    src = fetchFromGitHub {
+      owner = "ucscGenomeBrowser";
+      repo = pname;
+      rev = "v${version}_base";
+      sha256 = "1455dwzpaq4hyhcqj3fpwgq5a39kp46qarfbr6ms6l2lz583r083";
+    };
+    patches = [
+      ./kent-utils.patch
+      (fetchpatch {
+        url = "https://github.com/ucscGenomeBrowser/kent/commit/316e4fd40f53c96850128fd65097a42623d1e736.patch";
+        sha256 = "sha256-wr3NP5qoSonKz1TLKtQyrTPErCOk2gC1RimcX0tE7cM=";
+      })
+    ];
+  });
+in
+
+buildPerlModule rec {
+  pname = "Bio-BigFile";
+  version = "1.07";
+
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/L/LD/LDS/Bio-BigFile-${version}.tar.gz";
+    sha256 = "277b66ce8acbdd52399e2c5a0cf4e3bd5c74c12b94877cd383d0c4c97740d16d";
+  };
+
+  buildInputs = [ BioPerl IOString libpng libmysqlclient openssl ];
+
+  perlPreHook = ''
+    export KENT_SRC=${kent335}/
+    # Starting with GCC-11, compilation fails without -Wno-format-security
+    export CFLAGS=-Wno-format-security
```

Comment:
I think the classic way to do that is with `hardeningDisable`, as explained [here](https://nixos.org/manual/nixpkgs/unstable/#sec-hardening-in-nixpkgs).